src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -4144,8 +4144,8 @@ namespace Microsoft.FSharp.Core
         let inline ParseUInt16 (s:string)     = (# "conv.ovf.u2" (ParseUInt32 s) : uint16 #)
         let inline ParseIntPtr (s:string)  = (# "conv.ovf.i"  (ParseInt64 s)  : nativeint #)
         let inline ParseUIntPtr (s:string) = (# "conv.ovf.u"  (ParseInt64 s)  : unativeint #)
-        let inline ParseDouble (s:string)   = Double.Parse(s,NumberStyles.Float, CultureInfo.InvariantCulture)
-        let inline ParseSingle (s:string) = Single.Parse(s,NumberStyles.Float, CultureInfo.InvariantCulture)
+        let inline ParseDouble (s:string)   = Double.Parse(s.Replace("_", ""),NumberStyles.Float, CultureInfo.InvariantCulture)
```

Comment:
Why in the other places there is a trim?
As I see in the parser there are no leading or trailing whitespaces in numeric literals